packages/scandipwa/src/component/ContactForm/ContactForm.component.js

```diff
@@ -52,9 +53,27 @@ export class ContactForm extends FieldForm {
 
     onFormSuccess(fields) {
         const { onFormSubmit } = this.props;
-        onFormSubmit(fields);
+        onFormSubmit(fields).then(this.clearForm);
     }
 
+    clearForm = () => {
+        const {
+            form: {
+                form
+            } = {}
+        } = this;
+
+        if (!form) {
+            return;
+        }
+
+        const fields = form.querySelectorAll('input, textarea');
+        fields.forEach((input) => {
+            // eslint-disable-next-line no-param-reassign
+            input.value = '';
+        });
+    };
+
```

Comment:
Please don't use arrow functions as class methods. They are not extendable.
On the project, that is using SPWA theme, devs will need to copy entire arrow function in order to override it instead of calling parent method.